no_mutations.md

```diff
@@ -11,20 +11,266 @@ kernelspec:
   name: python3
 ---
 
+```{currentmodule} tskit
+```
+
 (sec_tskit_no_mutations)=
 
-# _Do you really need mutations?_
-% remove underscores in title when tutorial is complete or near-complete
+# Do you really need mutations?
+
+In tree sequences, the genetic genealogy exists independently of the mutations that
+generate genetic variation. This tutorial aims to illustrate
+cases where the genealogy on its own may be sufficent; a number of
+analyses do not actually require genetic variation.
+
+This applies particularly to cases where you generate tree sequences by simulation.
+In this case, although simulating variable sites allows you to generate actual genetic
+data, it's worth considering the following points:
```

Comment:
```suggestion
In simulations we know the true genealogies, and so it can be very helpful to work
with these directly. 
In real data, we might infer the trees and then work with the resulting genealogies.
(Of course, mutations add additional noise, and would be necessary
to produce data directly comparable to sequencing data.)
If you're wondering whether you need to add mutations at all,
it's worth considering the following points:
```